src/mozanalysis/experiment.py

```diff
@@ -506,29 +510,28 @@ def _build_enrollments_query_fenix_baseline(self, time_limits):
         # Try to ignore users who enrolled early - but only consider a
         # 7 day window
         return """
-        SELECT
-            b.client_info.client_id AS client_id,
-            mozfun.map.get_key(
-                b.ping_info.experiments,
-                '{experiment_slug}'
-            ).branch,
-            DATE(MIN(b.submission_timestamp)) AS enrollment_date
-        FROM `moz-fx-data-shared-prod.{dataset}.baseline` b
-        WHERE
-            DATE(b.submission_timestamp)
+            SELECT events.client_info.client_id AS client_id,
+                mozfun.map.get_key(
+                    e.extra,
+                    'branch'
+                ) AS branch,
+                DATE(MIN(events.submission_timestamp)) AS enrollment_date
+            FROM `moz-fx-data-shared-prod.{dataset}.events` events,
```

Comment:
I think the baseline/fallback method shouldn't change—it's intentionally using the baseline ping for cases where events aren't available. I'm not sure it will remain useful but I don't think we should change its behavior. I think there needs to be a separate method for Nimbus events.

Reply:
Added the fenix-fallback query back